#!/usr/bin/env python
PACKAGE = "control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


# only declaration, this is not the place to set the default values!
# look into params/lateral_controller instead!

# lateral control

gen.add("epsilon_division_by_zero",                       double_t, 0, "avoiding division by zero",                                     0,    0, 1)
gen.add("dist_k",                       double_t, 0, "distance weight",                                     0,    0, 10)
gen.add("lookahead_delta",              double_t, 0, "Weight for distance to look ahead for steering",      0, 0, 1)


exit(gen.generate(PACKAGE, "lateral_controller", "LateralController"))
 
